racing/db/queries.go

```diff
@@ -13,7 +13,9 @@ func getRaceQueries() map[string]string {
 				name, 
 				number, 
 				visible, 
-				advertised_start_time 
+				advertised_start_time,
+				CASE WHEN strftime('%s', advertised_start_time) >= strftime('%s','now')
```

Comment:
A side effect of this is all results will be closed given they have `advertised_start_date`'s all in the past. I e2e tested this by swapping out `'now'` for a date in the past temporarily.